### ISSUE TITLE
Adding evenfd inside polly

### ISSUE THREAD

**Issue by craff** [lindig/polly#2]
Could it be possible to add binding for
`̀``
#include <sys/eventfd.h>

int eventfd(unsigned int initval, int flags);
```
This is only one function, frequently used to wait on mutex/semaphore with epoll. So a binding for just that may be a bit overkill ?

**Comment by craff** [comment]
If you agree I could submit a PR.

**Comment by lindig** [comment]
Sorry to have missed this. Please send a PR . I noticed that you are using polly for your http server.

**Comment by craff** [comment]
Yes polly and event fd are the key to the performance of simple_httpd. I did an untested PR...

**Comment by lindig** [comment]
I have added this in 0.4.1 based on your PR with a few small changes. I am currently submitting this to Opam.

**Comment by craff** [comment]
Hello,

Christian Lindig ***@***.***> writes:

> I have added this in 0.4.1 based on your PR with a few small changes. I am currently submitting this to Opam.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.

Thanks, super fast ... I will use eventfd from polly and remove some
lines of code!

I submitted another issue about stub optimisation...

Cheers,
Christophe

--
Christophe Raffalli
tél: +689 87 23 11 48
web: http://raffalli.eu

Ce mail est signé avec pgp (Pièce jointe signature.asc, clef sur https://raffalli.eu/pgp)
This mail is signed with pgp (Attachment signature.asc, key  on  https://raffalli.eu/pgp)
